src/MIDebugEngine/Engine.Impl/Breakpoints.cs

```diff
@@ -192,9 +192,16 @@ private static BindResult EvalBindResult(Results bindResult, AD7PendingBreakpoin
             Debug.Assert(bkpt.FindString("type") == "breakpoint");
 
             string number = bkpt.FindString("number");
+            string warning = bkpt.TryFindString("warning");
             string addr = bkpt.TryFindString("addr");
 
-            PendingBreakpoint bp = new PendingBreakpoint(pbreak, number, StringToBreakpointState(addr));
+            PendingBreakpoint bp;
+            if (!string.IsNullOrEmpty(warning))
```

Comment:
warning is used for various things, such as symbols not loaded, checksum mismatch, condition can't be evaluated, etc. In the case of the vs debugger, we will never have a warning AND a bound breakpoint. I left the state as pending because it should be fine.